commit 9091939f22386527fa811014798a2be71d0f0788 (HEAD -> main)
Author: John Ott <john@johnott.us>
Date:   Thu Jan 11 22:14:59 2024 -0800

    enable building against apple_sdk_11_0

diff --git a/sources/ioregistry.m b/sources/ioregistry.m
index 5b8cd8c..ef18cc5 100644
--- a/sources/ioregistry.m
+++ b/sources/ioregistry.m
@@ -33,7 +33,7 @@ CGDisplayCount getOnlineDisplayInfos(DisplayInfos* displayInfos) {
         currDisplay->ioLocation = CFDictionaryGetValue(displayInfos, CFSTR("IODisplayLocation"));

         // Retrieving IORegistry entry for display
-        currDisplay->adapter = IORegistryEntryCopyFromPath(kIOMainPortDefault, (CFStringRef)currDisplay->ioLocation);
+        currDisplay->adapter = IORegistryEntryCopyFromPath(kIOMasterPortDefault, (CFStringRef)currDisplay->ioLocation);
         if (currDisplay->adapter == MACH_PORT_NULL) {
             continue;
         }
@@ -134,7 +134,7 @@ DisplayInfos* selectDisplay(DisplayInfos *displays, int connectedDisplays, char
 }

 static kern_return_t getIORegistryRootIterator(io_iterator_t* iter) {
-    io_registry_entry_t root = IORegistryGetRootEntry(kIOMainPortDefault);
+    io_registry_entry_t root = IORegistryGetRootEntry(kIOMasterPortDefault);
     kern_return_t ret = IORegistryEntryCreateIterator(root, kIOServicePlane, kIORegistryIterateRecursively, iter);
     if (ret != KERN_SUCCESS) {
         IOObjectRelease(*iter);
